spec/models/expense_spec.rb

```diff
@@ -16,8 +16,8 @@
       expect(expense).to be_valid
     end
 
-    it 'must have a value' do
-      expense = build(:expense, value: nil)
+    it 'must have an amount' do
+      expense = build(:expense, amount: nil)
 
       expect(expense).not_to be_valid
```

Comment:
This could be `expect(expense).to be_invalid`, which I think is nicer, since it is phrased in the positive.